harness/determined/_train_context.py

```diff
@@ -18,6 +18,14 @@ def __init__(self, env: det.EnvContext, hvd_config: horovod.HorovodContext):
         self.distributed = DistributedContext(env, hvd_config)
         self._stop_requested = False
 
+    @classmethod
+    def from_config(cls, config: Dict[str, Any]) -> "_TrainContext":
```

Comment:
non-blocking: prolly should add a docstring

Reply:
done